command.go

```diff
@@ -142,6 +146,10 @@ func (c *Command) slurpAllOutput() (bool, error) {
 
 	response, err := c.client.sendRequest(request)
 	if err != nil {
+		if errWithTimeout, ok := err.(errWithTimeout); ok && errWithTimeout.Timeout() {
```

Comment:
I'm wondering if we shouldn't instead use `url.Error` instead of declaring our own interface, and using `errors.As`:

```suggestion
		var errWithTimeout url.Error
		if errors.As(err, &errWithTimeout) && errWithTimeout.Timeout() {
```

This seems to be more idiomatic.

Reply:
Seems to do the same thing yes